src/main/java/org/jabref/logic/l10n/Localization.java

```diff
@@ -51,8 +51,7 @@ private Localization() {
      */
     public static String lang(String key, String... params) {
         if (localizedMessages == null) {
-            // I'm logging this because it should never happen
-            LOGGER.error("Messages are not initialized before accessing key: " + key);
+            // This will happen when running junit tests
```

Comment:
Why is this now a problem? While changing some localization-based stuff, I hit this error message sometimes during development and it was helpful to locate the problem. Hence, I would prefer if it could stay.

Reply:
Yes, but the problem now is that you will hit a NotInitlaizedException in junit. Execute the isbn fetcher test for example and you will see it. It has somehow to do with the fact that the log4j tries to call another appender (GUI based in the end, though it's executed from the test).

Reply:
The appender are/can be configured separately for the tests, so removing the GUI-appender is worth a try.

Reply:
Fixed the loging configuration, the xml file had to be renamed